src/Api/Tools/Models/Response/MemberAccessReportModel.cs

```diff
@@ -0,0 +1,29 @@
+﻿namespace Bit.Api.Tools.Models.Response;
+
+public class MemberAccessReportAccessDetails
+{
+    public Guid CollectionId { get; set; }
+    public Guid? GroupId { get; set; }
+    public string GroupName { get; set; }
+    public string CollectionName { get; set; }
+    public int ItemCount { get; set; }
+    public bool ReadOnly { get; set; }
+    public bool HidePasswords { get; set; }
+    public bool Manage { get; set; }
+
+    // internal to not expose 
+    internal ICollection<Guid> CipherIds { get; set; }
+    internal Guid? UserGuid { get; set; }
```

Comment:
Is this a TODO? If we don't want to expose these to the user then we shouldn't put them in the response model.

In both cases, I think you could track them outside the response model (e.g. in a local var or dict).